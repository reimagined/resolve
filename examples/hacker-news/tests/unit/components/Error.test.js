import React from 'react'
import { shallow } from 'enzyme'

import Error from '../../../client/components/Error'

it('renders correctly', () => {
  const markup = shallow(<Error location={{ search: '?text=SomeText' }} />)

  expect(markup).toMatchSnapshot()
})
