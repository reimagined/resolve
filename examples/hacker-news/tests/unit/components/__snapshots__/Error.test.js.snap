// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Error
    location={
      Object {
        "search": "?text=SomeText",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Error
        </h1>,
        "SomeText",
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Error",
        },
        "ref": null,
        "rendered": "Error",
        "type": "h1",
      },
      "SomeText",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Error
          </h1>,
          "SomeText",
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Error",
          },
          "ref": null,
          "rendered": "Error",
          "type": "h1",
        },
        "SomeText",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
